fix Raspberry Pi installation documentation:Fix raspberry pi documentation. (#74)
AOSC-Dev/wiki: content/aosc-os/devices/raspberrypi/4b/installation.md

- Raspberry Pi 3th gen boards (BCM2837A0/B0)
  - Raspberry Pi 3/3B/3B+
  - Compute Module 3

> - Some BCM2837 based Raspberry Pi 2 boards (2B Rev1.2,maybe slow)

Reviewer: ```suggestion

```

Let's just remove Pi 2 from the supported list.

# Known Problems

> - Hardware decoding is not working on 64-bit systems,see [here](https://github.com/raspberrypi/userland/blob/9f3f9054a692e53b60fca54221a402414e030335/CMakeLists.txt#L11) and [here](https://www.raspberrypi.org/forums/viewtopic.php?t=232684&start=25)

Reviewer: ```suggestion
- Hardware decoding is not working on 64-bit systems, see [here](https://github.com/raspberrypi/userland/blob/9f3f9054a692e53b60fca54221a402414e030335/CMakeLists.txt#L11) and [here](https://www.raspberrypi.org/forums/viewtopic.php?t=232684&start=25)
```

You are missing a space after the punctuation.

> - The raspi-config tool provided in the aosc package list was come from [cyanberry-config](https://github.com/AOSC-Archive/cyanberry-config),but now it is archived,and not suggested to use.

Reviewer: ```suggestion
- The `raspi-config` tool previously available in the AOSC package repository came from [cyanberry-config](https://github.com/AOSC-Archive/cyanberry-config). This project is now archived and no longer recommended to use.
```

Ditto. (Don't forget to modify the Chinese version accordingly.)

- Raspberry Pi 3th gen boards (BCM2837A0/B0)
  - Raspberry Pi 3/3B/3B+
  - Compute Module 3
- Some BCM2837 based Raspberry Pi 2 boards (2B Rev1.2,maybe slow)

# Known Problems

> - Hardware Decoding is not working on ARM64 architecture,see [here](https://github.com/raspberrypi/userland/blob/9f3f9054a692e53b60fca54221a402414e030335/CMakeLists.txt#L11) and [here](https://www.raspberrypi.org/forums/viewtopic.php?t=232684&start=25)

Reviewer: ```suggestion
- Hardware decoding is not working on 64-bit systems, see [here](https://github.com/raspberrypi/userland/blob/9f3f9054a692e53b60fca54221a402414e030335/CMakeLists.txt#L11) and [here](https://www.raspberrypi.org/forums/viewtopic.php?t=232684&start=25) for details.
```

Author: fixed.

To get your Pi running, a set of BSPs (Board Support Packages) must be installed upon the installation.

> Now the BSPs are in the main repo,you could just
> ```sh
> apt update
> ```
> 
> and
> 
> ```sh
> apt install linux+kernel+rpi64+lts
> ```

Reviewer: ```suggestion
The BSP packages are now present in the main suite, so you don't have to add `bsp-rpi` suite to your `sources.list`. Simply run the following command to install the BSP packages:

```sh
apt update
apt install linux+kernel+rpi64+lts
```

I think rewriting the section like that would be better.
